gui/src/widgets/menuspinbox.cpp

```diff
@@ -251,6 +273,12 @@ void MenuSpinbox::userInput(QString s)
 
 void MenuSpinbox::populateWidgets()
 {
+	// TODO: Review this function
+	if(!m_scalingEnabled) {
+		m_edit->setText(QString::number(m_value));
```

Comment:
do we need to setup signal blockers before modifying the value of the edit box?

Reply:
We might need to, good catch :sweat_smile: